### DISCUSSION TITLE
Moving to a dedicated communication tool (Matrix, Discord,...)

### DISCUSSION THREAD

**Discussion by Dynnammo** [ClaperCo/Claper#74] [Ideas]
Hey there. Thanks @alxlion for building Claper, it's great to see such project growing.

I wonder if it wouldn't be useful to begin to federate a community using a tool like Matrix/Discord.
I see many advantages (and drawbacks) of it :
- since Claper has about 300 stargazers, even gathering a tenth of it would be quite great in order to federate ideas and issues discussions
- IMHO developers aren't using the Discussions feature of GitHub, therefore using an external tool
- thoughts and ideas that need to be refined before beeing posted on GH

Drawbacks:
- moderation and minimal animation of the community. I could give an help on it.

**Comment by alxlion** [comment]
Hey ! There is already a Discord, the link is on the website: https://claper.co 😉
